cairo/ethereum/crypto/hash.cairo

```diff
@@ -23,3 +24,16 @@ func keccak256{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: Keccak
     tempvar hash = Hash32(value=value);
     return hash;
 }
+
+func Hash32__eq__(a: Hash32, b: Hash32) -> bool {
```

Comment:
just call `Bytes32__eq__` it's the same type

Reply:
didn't we want to differentiate between types?

Reply:
A hash32 is an alias to a bytes32